assets/styles/style.css

```diff
@@ -0,0 +1,256 @@
+body {
+  font-family:
+    Karma,
+    serif;
+  font-size: 16px;
+  width: 100%;
+  max-width: 100%;
+  background-color: white;
+  margin: auto;
+  margin-bottom: 6vh;
+  align-items: center;
+  justify-content: center;
+  line-height: 1.5;
+}
+
+/* main content */
+
+.main-content {
+  width: 64%;
+  position: relative;
+  align-items: center;
+  align-content: center;
+  margin-left: 18%;
+  margin-right: 18%;
+}
+
+.top-menu {
+  background-color: white;
+  width: 64%;
+  font-size: 24px;
+  height: 6vh;
+  display: flex;
+  align-items: center;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  justify-content: space-around;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding: 100px 0 0;
+}
+
+.images {
+  width: 15vw;
+  background-position: center;
+  background-size: cover;
+  position: relative;
+  flex-direction: column;
+  text-align: center;
+}
+
+img {
+  width: 15vw;
+}
+
+.bar {
+  text-align: center;
+  padding: 4vh 0 3vh;
+}
+
+.bar-item {
+  color: black;
+  background-color: white;
+  padding: 8px 16px;
+}
+
+.bar-item1 {
+  background-color: black;
+  color: white;
+  padding: 8px 18px;
+}
+
+.bar-item:hover {
+  background-color: black;
+  color: white;
+}
+
+.bar-item1:hover {
+  color: black;
+  background-color: #d8d6d6;
+}
+
+.line {
+  border: 0;
+  border-top: 1px solid #eee;
+  margin: 50px 0;
+}
+
+.about-me {
+  text-align: center;
+}
+
+.about-me h3 {
+  font-size: 26px;
+}
+
+.image-man {
+  width: 48vw;
+}
+
+.about-me h5 {
+  /* font-family: italic; */
+  font-size: 16px;
+}
+
+.about-me p {
+  width: 60vw;
+  margin: 0 2vw;
+}
+
+.footrow {
+  display: flex;
+  flex-flow: row wrap;
+  width: 64vw;
+  position: relative;
+  margin-left: 18%;
+  margin-right: 18%;
+  justify-content: left;
+  text-align: left;
+}
+
+.footer {
+  width: 20vw;
+  padding: 0 10px;
+}
+
+.footer h3 {
+  font-size: 24px;
+}
+
+.footer a {
+  color: black;
+}
```

Comment:
Your footer is completely off and this will automatically penalize you on the mobile responsiveness. Make sure you arrange that